README.md

```diff
@@ -1,7 +1,7 @@
 NVENC and NvFBC patches for Nvidia drivers
 ==========================================
 
-![GitHub last commit](https://img.shields.io/github/last-commit/keylase/nvidia-patch.svg) ![Latest version](https://img.shields.io/badge/latest%20linux%20driver%20version-590.44.01-brightgreen.svg)
```

Comment:
This seems to be missing an update in the table. Can you please run `tools/readme-autogen/readme-autogen.py` and update the PR?